src/libraries/iKin/src/iKinFwd.cpp

```diff
@@ -1566,18 +1566,45 @@ iCubTorso::iCubTorso(const string &_type) : iKinLimb(string(""))
 /************************************************************************/
 void iCubTorso::allocate(const string &_type)
 {
+    double version;
+    size_t underscore = getType().find('_');
```

Comment:
Please revise to be consistent with the formatting of the file.
For example, no spaces around symbols `=`, `,` ...